player/command.c

```diff
@@ -2830,12 +2830,13 @@ static int mp_property_sub_pos(void *ctx, struct m_property *prop,
     return mp_property_generic_option(mpctx, prop, action, arg);
 }
 
-static int mp_property_sub_text(void *ctx, struct m_property *prop,
-                                int action, void *arg)
+/// helper function for mp_property_sub_text and mp_property_secondary_sub_text 
+static int get_sub_text(void *ctx, struct m_property *prop,
+                                int action, void *arg, int sub_index)
```

Comment:
Spacing. You can probably remove the comment too. It's pretty obvious what this does.